packages/frontend/core/src/components/blocksuite/block-suite-editor/lit-adaper.tsx

```diff
@@ -94,11 +99,15 @@ const usePatchSpecs = (page: Doc, shared: boolean, mode: DocMode) => {
     };
   }, [mode, page.collection, page.id]);
 
+  const [enableAI] = useState(
```

Comment:
is there any possibility that useState without set means nothing

Reply:
Just for read a constant value

equal with
```
const enableAI = useRef(xxx).current
```

Reply:
it's a one time reading, why useState, useRef, useXxx